### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by keshakaneria**
Info about the project including installation process and contribution part. (For issue PR #81 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 Some pre-requisites:
 
 1. Make sure you have Flutter installed (https://flutter.dev/docs/get-started/install)
 2. Try installing and experimenting with the development version on your phone
 3. Don't hesitate in asking questions, we'll be happy to help
-   
+
+Hoping for your creative contributions.
+
 All further guidelines can be found in the [documentation](docs/).
```
Comment (yashlamba) [inline README.md]: We can remove this line now, as the website has the info

File: README.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+Before contributing, we highly recommend introducing yourself and your ideas on [gitter](https://gitter.im/codEd-org/simulate). You all can solve our [issues](https://github.com/cod-ed/simulate/issues) or help us find new to make this application more interesting and smooth.
 
 Some pre-requisites:
 
 1. Make sure you have Flutter installed (https://flutter.dev/docs/get-started/install)
 2. Try installing and experimenting with the development version on your phone
 3. Don't hesitate in asking questions, we'll be happy to help
-   
+
+Hoping for your creative contributions.
```
Comment (yashlamba) [inline README.md]: Let's keep the readme as simple as possible and about the project. You can remove this line.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 [![Gitter](https://badges.gitter.im/codEd-org/simulate.svg)](https://gitter.im/codEd-org/simulate?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Collection of simulations and visualizations in a cross platform app made using Flutter.
+Welcome to Simulate! Simulate is a collection of simulations and visualizations from various domains like mathematics, physics, computer science etc. in a cross platform app made using Flutter. The outcomes of these simulations mostly are really smooth (thanks to Flutter) and extremely beautiful (thanks to mathematics/sciences behind it). You can take a peek here.
+New to Simulate? Search for more information here https://cod-ed.github.io/simulate/
 
 # Contributing
 
-Before contributing, we highly recommend introducing yourself and your ideas on [gitter](https://gitter.im/codEd-org/simulate).
+Before contributing, we highly recommend introducing yourself and your ideas on [gitter](https://gitter.im/codEd-org/simulate). You all can solve our [issues](https://github.com/cod-ed/simulate/issues) or help us find new to make this application more interesting and smooth.
```
Comment (yashlamba) [inline README.md]: I think mentioning a link to issues is bit obsolete in a README, though it can help on the website.

**Review by yashlamba — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (keshakaneria) [reply README.md]: Edited as per requested @yashlamba

**Review by yashlamba — APPROVED** [review]

**Pull request merged**